Lib/dis.py

```diff
@@ -427,21 +427,26 @@ def __str__(self):
 class Formatter:
 
     def __init__(self, file=None, lineno_width=0, offset_width=0, label_width=0,
-                       line_offset=0, show_caches=False):
+                 line_offset=0, show_caches=False, *, show_positions=False):
         """Create a Formatter
 
         *file* where to write the output
         *lineno_width* sets the width of the line number field (0 omits it)
         *offset_width* sets the width of the instruction offset field
         *label_width* sets the width of the label field
         *show_caches* is a boolean indicating whether to display cache lines
+        *show_positions* is a boolean indicating whether positions should be
+        reported instead of line numbers.
```

Comment:
```suggestion
        *show_positions* is a boolean indicating whether full positions should be
        reported instead of only the line numbers.
```